doc/srcdeps.yaml

```diff
@@ -85,12 +91,103 @@ sourcesDirectory: ~/.m2/srcdeps
 # Possible values: error, warn, info, debug, trace
 #
 # Optional, default: warn
+# Since 1.0
 verbosity: warn
 
+# `buildTimeout` - the maximal time duration a build of a source dependency can take. When this value is
+# overrun, the build is terminated forcibly and the outer build fails too. Note that this value can be
+# overriden at the repository level - see below.
+#
+# Valid values consist of a number followed by a time unit abbreviation. Valid time unit abbreviations are:
+# * ns - nanoseconds
+# * us - microseconds
+# * ms - milliseconds
+# * s  - seconds
+# * m  - minutes
+# * h  - hours
+# * d  - days
+#
+# Examples: 10s, 15m, 2h
+#
+# Optional, default: 0x8000000000000000ms which is Long.MAX_VALUE milliseconds
+# Since 2.0
+buildTimeout: 14d
```

Comment:
no :) and the text does not imply it (does it?). I did not want to put `buildTimeout: 0x8000000000000000ms` there because I thought it would not be practical default for the people to start with when they copy the file to their project.